packages/core/src/common/message-service-protocol.ts

```diff
@@ -56,6 +61,10 @@ export interface ProgressMessageOptions extends MessageOptions {
      * Default: `true`
      */
     readonly cancelable?: boolean;
+    /**
+     * Default: `notification`
+     */
+    readonly location?: 'scm' | 'window' | 'notification';
```

Comment:
What happens if `scm` is not included in the application?
I'm only wondering if it's appropriate to have `core` know about `scm`.

Reply:
good point!

Reply:
@vince-fugnitto I've added `scm` progress reporting. In the case when no event consumer is registered for that location, nothing should happen.